google-cxApi/resources/cljsjs/google-cxApi/common/google-cxApi.ext.js

```diff
@@ -0,0 +1,83 @@
+/***************************************************************************
+ * Retrieved on 2015-09-22 from
+ * https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/google_universal_analytics_api.js
```

Comment:
Is this comment still relevant? If you have written the externs yourself, there probably should be copyright notice to Closure authors.